### PR TITLE
Fix rustc build

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
It's in order to make https://github.com/rust-lang/rust/pull/43009 build successfully.

### CODE REVIEW

**Comment by Yamakaky** [comment]
What is this lint for, again?

**Comment by GuillaumeGomez** [comment]
To warn when a doc comment is unused by rustdoc. As simple as that.

File: src/error_chain.rs [hunk 1]
```diff
@@ -423,6 +424,7 @@ macro_rules! impl_extract_backtrace {
     ($error_name: ident
      $error_kind_name: ident
      $([$link_error_path: path, $(#[$meta_links: meta])*])*) => {
+        #[allow(unused_doc_comment)]
         fn extract_backtrace(e: &(::std::error::Error + Send + 'static))
```
Comment (Yamakaky) [inline src/error_chain.rs]: Hum, do you know why this one is needed? This item doesn't have a doc.

**Comment by Yamakaky** [comment]
Also, could you update the changelog?

File: src/error_chain.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/error_chain.rs]: But it can because of its meta and then we'll have the issue.

**Comment by GuillaumeGomez** [comment]
Sure.

**Comment by GuillaumeGomez** [comment]
Done.

**Comment by Yamakaky** [comment]
Thanks!

**Comment by GuillaumeGomez** [comment]
Can we make a small release (0.0.x) so I can update cargo dependency please?

**Comment by Yamakaky** [comment]
I'll see that in this evening CEST.

**Comment by GuillaumeGomez** [comment]
Any news? :)

**Comment by Yamakaky** [comment]
Right, on it ^^

**Comment by GuillaumeGomez** [comment]
Thanks! :)

**Comment by Yamakaky** [comment]
0.11.0-rc.1 published.

My bad, I should have checked: could you fix https://travis-ci.org/brson/error-chain/jobs/258678124#L322-L334?

**Comment by GuillaumeGomez** [comment]
Ah right, sorry. I was testing with my own rustc version so I didn't saw it. Sending the fix.

**Pull request merged**